jobsdb/jobsdb.go

```diff
@@ -863,6 +864,7 @@ func (jd *HandleT) workersAndAuxSetup() {
 
 	jd.logger.Infof("Connected to %s DB", jd.tablePrefix)
 
+	jd.statPreDropTableCount = stats.DefaultStats.NewStat("jobsdb.pre_drop_tables_count", stats.GaugeType)
```

Comment:
don't we need tags for table prefix?

Reply:
Yes. Use NewTaggedStat with table prefix as the tag.

Reply:
done.